meetings/api/models.py

```diff
@@ -14,26 +14,8 @@ class Submission(models.Model):
     title = models.CharField(max_length=100)
     contributors = models.ManyToManyField(User, blank=True)
     description = models.TextField()
-    conference = models.ForeignKey('conference')
+    conference = models.ForeignKey('conferences.Conference')
```

Comment:
I'm having problems  when trying to go to /conferences. Think it comes from this line.
![screen shot 2016-07-05 at 8 47 21 am](https://cloud.githubusercontent.com/assets/11620890/16585090/1cbbeee0-428d-11e6-87fc-85e1b3c2b7a0.png)